fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -72,7 +72,7 @@ def fit(self, X, y, **kwargs):
         :param y: The label vector
         :type y: numpy.ndarray, pandas.DataFrame, pandas.Series, or list
         """
-        _, y_train, A = _validate_and_reformat_input(X, y, **kwargs)
+        X, y_train, A = _validate_and_reformat_input(X, y, **kwargs)
 
```

Comment:
we don't want this. why is this change here?

Reply:
Removed this.